truss/templates/server/model_wrapper.py

```diff
@@ -368,3 +360,30 @@ def _signature_accepts_kwargs(signature: inspect.Signature) -> bool:
 
 def _elapsed_ms(since_micro_seconds: float) -> int:
     return int((time.perf_counter() - since_micro_seconds) * 1000)
+
+
+def _handle_exception():
+    logging.exception("Exception while running predict")
```

Comment:
does `logging.exception` auto-magically figure out that there is an exception and log the stocktrace? If so, maybe leave a comment because it's not at all obvious to me that's happening. I"m looking around to try and figure out where you're logging the debug info.